frontend/src/routes/marketLink.svelte

```diff
@@ -3,20 +3,69 @@
 	import Button from '$lib/components/ui/button/button.svelte';
 	import { websocket_api } from 'schema-js';
 	import type { Readable } from 'svelte/store';
+	import { writable } from 'svelte/store';
+	import { Star } from 'lucide-svelte';
 
 	export let market: Readable<websocket_api.IMarket>;
 	$: marketIdParam = Number($page.params.id);
 	$: closed = $market.closed;
+
+	const starredStore = writable(
+		localStorage.getItem(`is_starred_${Number($market.id)}`) === 'true'
+	);
+	$: starred = $starredStore;
+
+	function handleStarClick() {
+		starredStore.update((currentValue) => {
+			const newValue = !currentValue;
+			localStorage.setItem(`is_starred_${Number($market.id)}`, newValue ? 'true' : 'false');
+			return newValue;
+		});
+	}
+
+	let isHovering = false;
 </script>
 
-<li class:order-2={!closed} class:order-4={closed}>
+<li
+	class:order-2={!closed && starred}
+	class:order-3={!closed && !starred}
+	class:order-5={closed && starred}
+	class:order-6={closed && !starred}
+	class="flex items-center gap-2"
```

Comment:
I think maybe starred should be another category (that way starred closed markets will also be on top)